profile/profileedit/index.php

```diff
@@ -1,26 +1,27 @@
 <!DOCTYPE html>
```

Comment:
Also, I think you should call the directory path to this file /profile/edit. I think it's redundant to have profile/profileedit.